docs/choreolib/installation.md

```diff
@@ -1,10 +1,12 @@
-Install ChoreoLib to your robot project by copy pasting the following link into the vendor library installation dialog:
+# Installation Methods
 
-```
+## Vendor Dependencies
+
+```json
 https://sleipnirgroup.github.io/ChoreoLib/dep/ChoreoLib.json
```

Comment:
Do we want to change this URL now?

Reply:
idk what is it supposed to be

Reply:
https://sleipnirgroup.github.io/ChoreoLib/dep/ChoreoLib2025Beta.json